source/_docs/custom-upstream.md

```diff
@@ -165,31 +165,26 @@ When testing is complete, merge any outstanding pull requests into the master br
 <h4 class="info">Note</h4>
   <ul>
     <li>For private repositories, create a dedicated user with read-only access to the repository. The password should only contain alphanumeric characters.</li>
-    <li>For private GitLab repositories, please ensure the user has the "reporter" permissions level.</li>
+    <li>For private GitLab repositories, please ensure the user has the <a href="https://docs.gitlab.com/ce/user/permissions.html#permissions">"reporter" permissions level</a>.</li>
   </ul>
 </div>
 
 ## Submit Product Information
 
 After you have a distribution that works on Pantheon, [contact support](/docs/getting-support) with the subject: "Product submission for [organization]" and include the following information about the distribution:
 
-- Name
-- Upstream URL:
- - Private repository URLs must follow the pattern `https://username:password@host.tld/organization/repository-name.git` (Do not use an email address in place of the username). Tokens may be used instead of a username and password if supported by your repository hosting provider. For example, private repositories hosted with GitHub can use [personal access tokens](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) to replace `username:password@github.com` with `personal_access_token@github.com`.
- - Public repository URLs do not require the `username:password@` portion of the above example URL.
-- Upstream Branch (usually master)
-- If the upstream repository is public or private
-- Description (< 200 characters, basic HTML allowed)
-- Logo (70x80)
-- Default connection mode (SFTP or Git)
-- Product author name and URL
-- Supporting information (< 200 characters, basic HTML allowed) i.e. drupal.org issue queue, support contacts, website
-- Links to evidence of testing on Pantheon. This should include the testing site, test files within the repository, and test reports.
+- **Upstream Name**
+- **Description**: (Optional) Less than 200 characters, basic HTML allowed

```

Comment:
We don't want to encourage HTML anymore. I think we want to allow markdown but am not sure we actually do. Either way let's encourage plain text for new upstreams until we figure it out.